src/_data/LiveData.js

```diff
@@ -45,11 +45,14 @@ const subscribeToWatchlist = st => {
     api.subscribeToTicks(favs.toJS());
 };
 
-export const changeLanguage = ln => {
-    api.changeLanguage(ln);
-    api.getActiveSymbolsFull();
-    api.getAssetIndex();
-    api.getTradingTimes();
+export const changeLanguage = async (dispatch, ln) => {
+    api.events.on('authorize', response => response.error ?
```

Comment:
We want to use the promise + await approach in this case.
Though I am not sure even if this needs to be here?!?